packages/client/src/app/app.module.ts

```diff
@@ -105,7 +105,7 @@ const toolbarOptions = [
     NotificationsComponent, WatchingPageComponent, HistoryComponent, StartConversationComponent, TruncatePipe
   ],
   imports: [
-    BrowserModule, AppRoutingModule, HttpClientModule, FormsModule, ReactiveFormsModule, ToppyModule,
+    BrowserModule, AppRoutingModule, HttpClientModule, FormsModule, ReactiveFormsModule, CookieModule.forRoot(),
```

Comment:
should the CookieModule get a line of its own, to match the pattern we have down below (i.e. anything that does more than just get imported gets its own line)

Reply:
it should!